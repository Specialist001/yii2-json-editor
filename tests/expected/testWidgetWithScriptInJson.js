w6JsonEditor_a3048777 = new JSONEditor(document.getElementById('w6-json-editor'), {"mode":"tree","onChange":function() {jQuery('#w6').val(w6JsonEditor_a3048777.getText());}}, {"script":"\u003Cscript type=\u0022text\/javascript\u0022\u003Ealert(\u0022XSS\u0022);\u003C\/script\u003E"});
jQuery('#w6').parents('form').submit(function() {jQuery('#w6').val(w6JsonEditor_a3048777.getText());});