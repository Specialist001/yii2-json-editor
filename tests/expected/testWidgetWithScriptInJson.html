<input type="hidden" id="w6" name="data" value="{&quot;script&quot;: &quot;&lt;script type=\&quot;text/javascript\&quot;&gt;alert(\&quot;XSS\&quot;);&lt;/script&gt;&quot;}" data-json-editor-name="w6JsonEditor_a3048777"><div id="w6-json-editor" style="height: 250px;"></div>